pages/services/confluent-kafka/2.6.0-5.1.2/security/index.md

```diff
@@ -0,0 +1,259 @@
+
+#### Prerequisites
+- The hostname and port of a KDC reachable from your DC/OS cluster
+- Sufficient access to the KDC to create Kerberos principals
+- Sufficient access to the KDC to retrieve a keytab for the generated principals
+- [The DC/OS Enterprise CLI](/1.10/cli/enterprise-cli/#installing-the-dcos-enterprise-cli)
+- DC/OS Superuser permissions
+
+#### Configure Kerberos Authentication
+
+#### Create principals
+
+The DC/OS {{ model.techName }} service requires a Kerberos principal for each broker to be deployed. Each principal must be of the form
+```
+<service primary>/kafka-<broker index>-broker.<service subdomain>.autoip.dcos.thisdcos.directory@<service realm>
+```
+with:
+- `service primary = service.security.kerberos.primary`
+- `broker index = 0 up to brokers.count - 1`
+- `service subdomain = service.name with all `/`'s removed`
+- `service realm = service.security.kerberos.realm`
+
+For example, if installing with these options:
+```json
+{
+    "service": {
+        "name": "a/good/example",
+        "security": {
+            "kerberos": {
+                "primary": "example",
+                "realm": "EXAMPLE"
+            }
+        }
+    },
+    "brokers": {
+        "count": 3
+    }
+}
+```
+then the principals to create would be:
+```
+example/kafka-0-broker.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/kafka-1-broker.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/kafka-2-broker.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+```
+#include /services/include/security-kerberos-ad.tmpl
+
+#include /services/include/security-service-keytab.tmpl
+
+#### Install the Service
+
+Install the DC/OS {{ model.techName }} service with the following options in addition to your own:
+```json
+{
+    "service": {
+        "security": {
+            "kerberos": {
+                "enabled": true,
+                "enabled_for_zookeeper": <true|false default false>,
+                "kdc": {
+                    "hostname": "<kdc host>",
+                    "port": <kdc port>
+                },
+                "primary": "<service primary default kafka>",
+                "realm": "<realm>",
+                "keytab_secret": "<path to keytab secret>",
+                "debug": <true|false default false>
+            }
+        }
+    }
+}
+```
+
+<p class="message--note"><strong>NOTE: </strong>If <tt>service.kerberos.enabled_for_zookeeper</tt> is set to true, then the additional setting <tt>kafka.kafka_zookeeper_uri</tt> must be configured to point at a kerberized Confluent Zookeeper as shown below. </p>
+
+```json
+{
+    "kafka": {
+        "kafka_zookeeper_uri": <list of zookeeper hosts>
+    }
+}
+```
+The DC/OS {{ model.kafka.zookeeperTechName }} service (`{{ model.kafka.zookeeperPackageName }}` package) is intended for this purpose and supports Kerberos.
```

Comment:
Please confirm that this should refer to Kafka-Zookeeper, not Confluent-Kafka.

Reply:
A preview of how it will look like:
![https://i.imgur.com/iRSFZYP.png](https://i.imgur.com/iRSFZYP.png)

`model.kafka.zookeeperTechName` renders to Confluent Zookeeper here.